packages/prisma/schema.prisma

```diff
@@ -21,6 +26,7 @@ model User {
   password         String?
   source           String?
   signature        String?
+  roles            Role[]           @default([USER])
```

Comment:
Isn't `role` better than `roles`?

Reply:
since a user can have multiple roles, I thought `roles` is more appropriate.

Reply:
Yeah I think the idea of allowing for multiple role memberships in the future might be worthwhile. Admittedly we could make the change later but this saves headache 😄

Reply:
Yeahh